mule-migration-tool-engine/src/test/java/com/mulesoft/tools/migration/engine/MigrationJobTest.java

```diff
@@ -177,4 +178,18 @@ public void executeCheckApplicationModel() throws Exception {
                is(2));
 
   }
+
+  @Test
+  public void execute_assertGitIgnoreFile() throws Exception {
```

Comment:
Do you think it would be useful to check the content of the created files to make sure the .gitignore is created correctly?

Reply:
add further test, check contnet